README.md

```diff
@@ -58,6 +58,14 @@ An RFC is ready to be approved and finalized once it's Pull Request is ready for
 
 At this time, some member of the core team will motion for a final comment period (FCP). This follows our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process. Once the final comment period has elapsed the RFC will be merged if there are no objections.
 
+## RFC Meetings
+
+RFCs advance through the stages during RFC meetings with TSC and maintainers. Voting follows the [RFC proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
+
+Meetings occur ad hoc rather than on a scheduled basis. They are called for when a proposal author or champion feels it is ready to advance to the next stage. The author or champion can ask for a meeting by contacting TSC to schedule a time.
```

Comment:
Would doing these discussions during a Tuesday community call actually be a good idea? Would it increase participation/attendance (esp. from the rest of core), give @sarah11918 a built-in agenda item, and require less planning to organize and find a time?

Reply:
Especially reading the next paragraph, it feels a little "secret meeting" and the transparency wins of doing it on a public call could be worth it for that alone.

Reply:
We could, but some issues we had before when RFC meetings were open to all:

- Discussions tended to be very high-level, most of the audience didn't understand the technical details that are an important part.
- Discussions tended to veer towards bikeshedding often.
- It was always odd to me that we tried to reach consensus to the general community, which meant a non-maintainer could object and per the rules it then went to a TSC decision.